Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -108,7 +108,13 @@ def Ordinal : Type (u + 1) :=
 /-- A "canonical" type order-isomorphic to the ordinal `o`, living in the same universe. This is
 defined through the axiom of choice.
 
-Use this over `Iio o` only when it is paramount to have a `Type u` rather than a `Type (u + 1)`. -/
+Use this over `Iio o` only when it is paramount to have a `Type u` rather than a `Type (u + 1)`,
+and convert using
+```
```

Comment:
Nit: I personally find it more legible to leave a space between text and code blocks. (GitHub won't let me suggest this for syntax reasons)